client/ayon_houdini/api/usd.py

```diff
@@ -205,8 +208,8 @@ def get_configured_save_layers(usd_rop, strip_above_layer_break=True):
     lop_node = get_usd_rop_loppath(usd_rop)
     stage = lop_node.stage(apply_viewport_overrides=False)
     if not stage:
-        raise RuntimeError(
-            "No valid USD stage for ROP node: " "%s" % usd_rop.path()
+        raise PublishError(
+            f"No valid USD stage for ROP node: '{usd_rop.path()}' "
```

Comment:
This should not be a publish error - the previous type was better. This may be called outside of publishing if needed.

Reply:
I think that is the case already. I've reverted the changes in `usd.py`